src/step2-index.js

```diff
@@ -1,3 +1,7 @@
+import './web//css/style.css';
```

Comment:
style.css를 분리하는 게 좋겠다는 생각이 들어 여러 레퍼런스를 찾아봤는데
하나의 css파일을 여러 개로 분리하는 것에 대해서는 찾아보기가 힘들고
찾아봐도 사람마다 기준이 달라 저만의 기준이 만들어지기 전까지는 하나로 유지하는 것으로 결정했습니다!
개인적으로는 각 container별로 나누는 것이 괜찮겠다는 생각이 들면서도, 그러면 공통적으로 들어가는 button이나 number input 부분, class로 선언된 것들을 어디로 나누어야 할지에 대해서도 고민중이에요.

Reply:
네 나쁘지 않은 방향인 듯 합니다. 스타일 코드도 다른 곳과 비슷한 맥락으로 나뉠 수 있을 것 같은데 기본적으로는 아래와 같은 영역을 먼저 분리해본 뒤 나머지 스타일들은 컨테이너 단위로 관리해도 괜찮지 않을까 싶어요.

- global style
- reset / normalize style
- common / util style

추가로 file path가 정확한지 확인 부탁드려요~

```suggestion
import './web/css/style.css';
```